tests/test_quixstreams/test_dataframe/test_registry.py

```diff
@@ -0,0 +1,29 @@
+from unittest.mock import MagicMock, PropertyMock
+
+import pytest
+
+from quixstreams.dataframe import DataFrameRegistry, StreamingDataFrame
+from quixstreams.models import Topic
+
+
+class TestDataFrameRegistry:
+    def test_register_root_multi_topic_sdf_fails(self):
+        registry = DataFrameRegistry()
+        sdf_mock = MagicMock(spec_set=StreamingDataFrame)
+        type(sdf_mock).topics = PropertyMock(
+            return_value=[Topic("test-1"), Topic("test-1")]
+        )
```

Comment:
This may be simplified
```suggestion
        sdf_mock = mock.Mock(topics=[Topic("test-1"), Topic("test-1")])
```